test/OpenTelemetry.Extensions.Hosting.Tests/OpenTelemetry.Extensions.Hosting.Tests.csproj

```diff
@@ -31,7 +31,8 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="6.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
-    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.0" />
+    <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="7.0.0-preview.6.22330.3" />
```

Comment:
Can this be put it in https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/Common.nonprod.props#L28?

Reply:
we use multiple versions based on tfm.

Reply:
We could centralize the versions in Common.nonprod.props if we defined a property for each version. Something like `MicrosoftAspNetCoreTestPkgVer6` and `MicrosoftAspNetCoreTestPkgVer7`.

We have at least [one more project](https://github.com/open-telemetry/opentelemetry-dotnet/blob/aba6eafb8f62614f4035e280bef77d07c21c569a/test/OpenTelemetry.Exporter.Prometheus.AspNetCore.Tests/OpenTelemetry.Exporter.Prometheus.AspNetCore.Tests.csproj#L28) that uses the `TestHost` package.

Also, the AspNetCore.Tests project uses a different test package (Microsoft.AspNetCore.Mvc.Testing) but it looks like it is versioned in sync with the `TestHost` package.
* https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing#versions-body-tab
* https://www.nuget.org/packages/Microsoft.AspNetCore.TestHost#versions-body-tab